addons/l10n_be/data/template/account.account-be.csv

```diff
@@ -53,18 +53,18 @@
 "a240000","Furniture & material","240000","asset_fixed","","False","Meubles et matériel","Meubilair & materiaal","Möbel & Material","Large expenses related to workplaces","attn_VAT-IN-V83-21"
 "a240008","Furniture and Vehicles - Capital Profits - Recorded","240008","asset_fixed","","False","Mobilier et matériel roulant - Plus-values actées","Meubilair en rollend materieel - Geboekte meerwaarden","Geschäftsausstattung und Fuhrpark - Gebuchte Mehrwerte","",""
 "a240009","Furniture and Vehicles - Depreciations and Amounts Written Down - Recorded","240009","asset_fixed","","False","Mobilier et matériel roulant - Amortissements ou moins-values actés","Meubilair en rollend materieel - Geboekte afschrijvingen of waardeverminderingen","Geschäftsausstattung und Fuhrpark - Gebuchte Abschreibungen oder Wertminderungen","",""
-"a241","Purchase (cars)","241","asset_fixed","","False","Achat (voitures)","Aankoop (auto's)","Kauf (Autos)","Purchase of a car, including second hand or after-lease contract",""
+"a241","Purchase (cars)","241","asset_fixed","","False","Achat (voitures)","Aankoop (auto's)","Kauf (Autos)","Purchase of a car, including second hand or after-lease contract","attn_VAT-IN-21-D"
 "a241001","Purchase (motorbikes)","241001","asset_fixed","","False","Achat (motos)","Aankoop (motoren)","Kauf (Motorräder)","Purchase of a motorbike, including second hand or after-lease contract","attn_VAT-IN-V83-21"
 "a241002","Purchase (bikes)","241002","asset_fixed","","False","Achat (vélos)","Aankoop (fietsen)","Kauf (Fahrräder)","Purchase of a bike, including second hand","attn_VAT-IN-V83-21"
 "a241003","Purchase (e-bikes)","241003","asset_fixed","","False","Achat (vélos électriques)","Aankoop (e-bikes)","Kauf (E-Bikes)","Purchase of a e-bike, including second hand","attn_VAT-IN-V83-21"
 "a241004","Purchase (utility cars)","241004","asset_fixed","","False","Achat (voitures utilitaires)","Aankoop (bedrijfsauto's)","Kauf (Nutzfahrzeuge)","Purchase of a utility car, including second hand or after-lease contract","attn_VAT-IN-V83-21"
-"a2411","Accessories (cars)","2411","asset_fixed","","False","Accessoires (voitures)","Accessoires (auto's)","Zubehör (Autos)","All accessories purchases related to your professional vehicle",""
+"a2411","Accessories (cars)","2411","asset_fixed","","False","Accessoires (voitures)","Accessoires (auto's)","Zubehör (Autos)","All accessories purchases related to your professional vehicle","attn_VAT-IN-21-D"
```

Comment:
Same, should be an IG tax as well (asset account) and same question about deductibility @chklop